pytripgui/controller/plan_cont.py

```diff
@@ -39,7 +42,11 @@ def new_plan(model):
 
         # open a dialog for the user to edit it
         PlanController.edit_plan(model, plan)
+
+        logger.debug("model.plans.append(plan)")
         model.plans.append(plan)
+        logger.debug(plan)
+        # TODO: TreeController.update_tree()
```

Comment:
I need some suggestions here, what is the best way to comminuate to TreeController?